### DISCUSSION TITLE
Profile privacy

### DISCUSSION THREAD

**Discussion by Dm1triyDen1senko** [community/community#73713] [New to GitHub]
Hi!

I have a problem - it is impossible to see my profile via link despite the "private" mode is disabled...

Could somebody helps me please?

**Comment by OS-Scratch** [comment]
Go to [profile picture] -> Settings -> Public profile -> then scroll down until you see something like this: 
![image](https://github.com/user-attachments/assets/a7057f04-0cd8-47c9-99f5-c46568fe45b9)
Disable private mode and click on "Update Preferences".

**Comment by IzonIcy** [comment]
this happend to me i think you have to make your portfolio public and then come back and try to do it

**Comment by OS-Scratch** [comment]
There are some solutions for the link problem: Log out and try viewing your profile. Can you look at it? If not, verify that the link is github.com/Dm1triyDen1senko.

**Comment by IzonIcy** [comment]
It is working now

On Wed, Jan 8, 2025, 10:44 AM OS-Scratch ***@***.***> wrote:

> There are some solutions for the link problem: Log out and try viewing
> your profile. Can you look at it? If not, verify that the link is
> github.com/Dm1triyDen1senko.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/73713#discussioncomment-11777203>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLP65SCHTNCSKLRC2DTKCWT2JVWXPAVCNFSM6AAAAABU2Q5RY2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNZXG4ZDAMY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

-- 
-----------
This is a student email account managed by the *Menlo Park City 
School District*. The contents of this email are governed by the laws of 
the State of California and the board policies of the school district.

**Reply by OS-Scratch** [reply]
Good!
